DESCRIPTION

```diff
@@ -40,8 +40,17 @@ Imports:
     shiny,
     shinyjs
 Suggests:
+    officedown,
+    officer,
     pkgload,
-    usethis
+    purrr,
+    rmarkdown,
+    rprojroot,
+    testthat,
+    tidyr,
+    usethis,
+    withr,
+    xml2
```

Comment:
Add all the packages used only in the review scripts/tests as Suggests, since they are not exposed as package functions.